packages/page-anchor/src/Check.tsx

```diff
@@ -21,12 +21,6 @@ type AnchorStatus = {
   tag: 'hashing',
 };
 
-/* function depends<T, R>(compute: (_: T) => Promise<R>, dat: T, initial: R): R {
```

Comment:
Removing dead code. This was a failed Idea that should have been removed in the original anchoring PR.